src/bm3dornl/bm3d.py

```diff
@@ -739,26 +739,46 @@ def bm3d_ring_artifact_removal_ms(
             filter_kwargs=filter_kwargs,
         )
 
-    # step 1: create a list of binned sinograms
-    binned_sinos = horizontal_binning(sinogram, k=k)
-    # reverse the list
-    binned_sinos = binned_sinos[::-1]
-
-    # step 2: estimate the noise level from the coarsest sinogram, then working back to the original sinogram
-    noise_estimate = None
-    for i in range(len(binned_sinos)):
-        logging.info(f"Processing binned sinogram {i+1} of {len(binned_sinos)}")
-        sino = binned_sinos[i]
-        sino_star = (
-            sino if i == 0 else sino - horizontal_debinning(noise_estimate, sino)
+    denoised_sino = None
+    # Make a copy of an original sinogram
+    sino_orig = horizontal_binning(sino_star, 1, dim=0)
+    binned_sinos_orig = [np.copy(sino_orig)]
+
+    # Contains upscaled denoised sinograms
+    binned_sinos = [np.zeros(0)]
```

Comment:
why creating an np.zeros(0) here? Are we trying to do `np.empty` or something else?